packages/cli/src/commands/lockedgold/withdraw.ts

```diff
@@ -16,6 +17,8 @@ export default class Withdraw extends BaseCommand {
   static examples = ['withdraw 3600']
 
   async run() {
+    await requireNodeIsSynced(this.web3)
```

Comment:
we should make this step part of base command so that first time we access web3 this get's checked... or on the command init.

If not, everytime a dev adds a command it need to remember to add it... and thus lead to human error